test/playwright/bfcache.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect } from "@playwright/test";
```

Comment:
bfcache is a small use case. I don’t want to add a huge tool for that.

It is better for me to test by hands this use case.

Reply:
I removed playwright and all tests